### DISCUSSION TITLE
binaries.prisma.sh not accessible via IPv6

### DISCUSSION THREAD

**Discussion by ldpr** [prisma/prisma#16763] [Q&A] [answered]
Example -> <https://dnschecker.org/#AAAA/binaries.prisma.sh>

Makes it unusable on IPv6 only machines.

**Comment by nurul3101** [comment, marked as answer]
Hey @ldpr 👋 
Can you create a [GitHub Issue](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that our engineering team can have a look and provide insights?

**Reply by janpio** [reply]
Yes please @ldpr - that is definitely something we want to fix. Thanks!

**Reply by ldpr** [reply]
@nurul3101 @janpio sorry for the crappy formatting, i just converted it to an issue here https://github.com/prisma/prisma/issues/16777

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
